src/tutorials/pet-shop.md

```diff
@@ -329,7 +329,7 @@ To test the `adopt()` function, recall that upon success it returns the given `p
    ```javascript
    // Testing the adopt() function
    function testUserCanAdoptPet() public {
-     uint returnedId = adoption.adopt(expectedPetId);
+     uint returnedId = uint(adoption.adopt(expectedPetId));
```

Comment:
Is this solving a problem? I was able to follow the tutorial and this bit caused no problem for me.